packages/git/src/browser/git-contribution.ts

```diff
@@ -976,12 +976,11 @@ export class GitAmendSupport implements ScmAmendSupport {
 
     constructor(protected readonly repository: Repository, protected readonly git: Git) { }
 
-    public async getIntialAmendingCommits(storedState: string, lastHead: string): Promise<ScmCommit[]> {
-        const { amendingHeadCommitSha } = JSON.parse(storedState);
+    public async getIntialAmendingCommits(amendingHeadCommitSha: string, latestCommitSha: string): Promise<ScmCommit[]> {
```

Comment:
FYI there is a typo on Initial

Reply:
Yes, thank you so much for noticing that as that function name will be API.  I have corrected it.